tests/ui_tools/test_boxes.py

```diff
@@ -1642,7 +1677,10 @@ def test_keypress_CYCLE_COMPOSE_FOCUS(
             else:
                 write_box.stream_box_view(stream_id)
         else:
-            write_box.private_box_view()
+            if recipient_box_editable:
+                write_box.private_box_view()
+            else:
+                write_box.private_box_edit_view()
```

Comment:
We can adjust this test as soon as we have these functions, ie. sooner in the sequence of commits and potentially in the commit that adds this method (can squash later, but keep it adjacent).